containers/daap-get-glue-metadata/src/var/task/get_glue_metadata.py

```diff
@@ -1,12 +1,24 @@
 import json
+import os
 
 import boto3
+from data_platform_logging import DataPlatformLogger
+
+logger = DataPlatformLogger(
+    extra={
+        "image_version": os.getenv("VERSION", "unknown"),
+        "base_image_version": os.getenv("BASE_VERSION", "unknown"),
+    }
+)
 
 
 def handler(event, context):
     database = event["queryStringParameters"]["database"]
     table = event["queryStringParameters"]["table"]
 
+    logger.info(f"database: {database}")
+    logger.info(f"table: {table}")
```

Comment:
should use this instead
```suggestion
    logger.add_extras(
        {
            "lambda_name": context.function_name,
            "data_product_name": database,
            "table_name": table
        }
    )